src/init.ts

```diff
@@ -266,7 +267,7 @@ export class NightwatchInit {
     }
 
     // Read outDir property from tsconfig.json file.
-    const tsConfig = JSON.parse(fs.readFileSync(tsConfigPath, 'utf-8'));
+    const tsConfig = JSON5.parse(fs.readFileSync(tsConfigPath, 'utf-8'));
```

Comment:
Should we replace all the `JSON.parse` with `JSON5.parse`?

Reply:
No, whenever json file has comments, we need to use JSON5.parse as JSON.parse is not handling when comments are present in json file.